### ISSUE TITLE
SystemManager: show process info in the main window 

### ISSUE THREAD

**Issue by diversys** [X547/HaikuUtils#4]
This way it is much easier to see what's going on and doesn't require extra clicks.

Example:

![image](https://user-images.githubusercontent.com/1316513/100342546-2fc41180-2fef-11eb-8093-db277b6a4b7f.png)

**Comment by X547** [comment]
Just for interest: what operating system is used in this screenshot? It doesn't look like windows or X11/Wayland based *nix.

**Comment by diversys** [comment]
https://github.com/SerenityOS/serenity

Here's a video where the developer implemented said feature:
https://www.youtube.com/watch?v=ofhjmHhzqAU

You might find interesting other videos of him working on Process Manager (now System Monitor) as well as many other parts of his OS :)

**Comment by X547** [comment]
My SystemManager is largely inspired by [ProcessHacker](https://github.com/processhacker/processhacker). But instead of one application, separate applications are used for processes, services, sockets (not yet published).

**Comment by diversys** [comment]
Yeah, I also think that separate apps are better and this app includes too many features. However, it still might be useful to borrow some ideas from. Like list of open files by a process. I don't think there are any apps that can do that on Haiku.

**Comment by X547** [comment]
> Like list of open files by a process.

This feature is already implemented in [Handles](https://github.com/X547/HaikuUtils/tree/master/Handles) utility and it is planned to be moved in Handles tab of team window (currently stubbed). Unfortunately Haiku kernel do not provide information about opened file names and paths so this information can't be displayed ([#15774](https://dev.haiku-os.org/ticket/15774)).

**Comment by diversys** [comment]
There is this, not sure how useful it is.

```
fdinfo --help
Usage: fdinfo <id/pattern> or -[dD] <path-to-device> or -[fF] <file>
  Shows info about the used file descriptors in the system.

  -d    Only shows accesses to the given device
  -D    Likewise, but only shows the teams that access it
  -f    Only shows accesses to the given file
  -F    Likewise, but only shows the teams that access it
```


```
fdinfo -D /bin/bash
 1690 /boot/system/servers/debug_server
 1770 /boot/system/Tracker
 1780 /boot/system/Deskbar

```

**Comment by X547** [comment]
> fdinfo -D /bin/bash

This works in opposite direction (file name -> team list). It can't be used to efficiently get file name of FD.
